.pre-commit-config.yaml

```diff
@@ -56,6 +56,18 @@ repos:
   hooks:
   - id: ruff
 
+- repo: https://github.com/asottile/pyupgrade
+  rev: v3.15.0
+  hooks:
+  - id: pyupgrade
+    args: [--py310-plus]
+
+- repo: https://github.com/adamchainz/django-upgrade
```

Comment:
Perhaps use the `files` option to only include the files in the `rocky/` folder?